src/main/java/eggventory/items/StockType.java

```diff
@@ -172,8 +187,53 @@ public String getName() {
         return name;
     }
 
-    public void setName(String name) {
-        this.name = name;
+    /**
+     * Updates the name of the StockType.
+     * @param newName String which uniquely identifies a StockType.
+     * @return The name of the StockType, for printing purpose.
+     */
+    public String setName(String newName) {
```

Comment:
the only place in this commit where i see you use setName (line 151 of stockList), you don't use the return value anyway so i'm not sure why you made it string

Reply:
Yeah this can be made void. Good catch.